src/components/navBar/navBar.spec.js

```diff
@@ -156,6 +156,17 @@ describe('mdNavBar', function() {
 
       expect($scope.selectedTabRoute).toBe('tab2');
     });
+
+    it('does not update tabs if tab controller is undefined', function() {
+      $scope.selectedTabRoute = 'tab1';
+
+      spyOn(Object.getPrototypeOf(ctrl), '_updateInkBarStyles');
```

Comment:
is `Object.getPrototypeOf` necessary? I'm surprised you can't just do `spyOn(ctrl, '_upateInkBarStyles');`